web/client/components/catalog/editor/AdvancedSettings/WMSDomainAliases.js

```diff
@@ -0,0 +1,76 @@
+import {FormControl, FormGroup, Col, ControlLabel, Glyphicon, Button} from "react-bootstrap";
+import {debounce, size, map, omit, toInteger} from "lodash";
+import Message from "../../../I18N/Message";
+import InfoPopover from "../../../widgets/widget/InfoPopover";
+import React, {useState, useCallback, useEffect} from "react";
+import tooltip from "../../../misc/enhancers/tooltip";
+
+/**
+ * @name WMSDomainAliases
+ * @param onChangeServiceProperty
+ * @param service {Object}
```

Comment:
a description is generally added after

and the types comes before the name of the param
```suggestion
 * @param {function} onChangeServiceProperty ...
 * @param {object} service ...
```

Reply:
Corrected it, thanks